### ISSUE TITLE
addCohortIntersect cannot be chained

### ISSUE THREAD

**Issue by KimLopezGuell** [oxford-pharmacoepi/CohortProfiles#64]
As it only outputs "cohort_definition_id", "subject_id", "cohort_start_date" and "cohort_end_date" plus the new column, if you chain addCohortIntersect calls, only the newest added column remains. Moreover, any other pre-existing column outside the previously mentioned four is removed (i.e. if we had "age" or "sex" added, for instance). We should not allow that, right? @catalamarti @ilovemane

**Comment by catalamarti** [comment]
Yes, it should be nice that we output everything. How I was working it was:
x : original table
```
x_new <- x %>%
  select("subject_id", "cohort_start_date") %>%
  distinct() %>%
  .... add new columns ....
return( x %>% right_join(x_new))
```
